api/dsService/dsModel.js

```diff
@@ -14,6 +14,24 @@ const postMeeting = (meeting) => {
   return dsClient.post(`/Meetings/create`, meeting);
 };
 
+//below chunk has been commented out for the linter... Once we have DS working, we can uncomment and utilize this code...
+
+/*
+//update meeting needs to be verified with DS... This may or may not work, but currently it can't be tested...
+//for now, we will just update the local db
+const updateMeeting = (meeting, id) => {
+  return dsClient.update(`/Meetings/${id}`, meeting);
+};
+
+
+//update meedings needs to be verified with DS... This may or may not work, but currently it can't be tested...
+//for now, we will just delete from the local db
+const deleteMeeting = (id) => {
+  return dsClient.delete(`/Meetings/${id}`);
+};
+
+*/
+
```

Comment:
I like how you commented out code for future use, great use of comments

Reply:
Thank you very much!